disnake/ui/select/base.py

```diff
@@ -24,19 +24,24 @@
 __all__ = ("BaseSelect",)
 
 if TYPE_CHECKING:
-    from typing_extensions import ParamSpec, Self
+    from typing_extensions import (
+        ParamSpec,
+        Self,
+        TypeVar,  # noqa: TC004
+    )
 
     from ...abc import Snowflake
     from ...interactions import MessageInteraction
     from ..item import ItemCallbackType
     from ..view import View
 
+    V_co = TypeVar("V_co", bound="Optional[View]", covariant=True, default=Optional[View])
 else:
     ParamSpec = TypeVar
+    V_co = TypeVar("V_co", bound="Optional[View]", covariant=True)
```

Comment:
https://github.com/DisnakeDev/disnake/pull/1441/files#r2443495659

```suggestion
    V_co = TypeVar("V_co")
```